charts/openshift-etcd-backup/Chart.yaml

```diff
@@ -21,9 +21,4 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
       description: |
-        upgrade openshift-etcd-backup image to v1.9.6
-
-        * fixed prometheus alert
-      links:
-        - name: openshift-etcd-backup release v1.9.6
-          url: https://github.com/adfinis/openshift-etcd-backup/releases/tag/v1.9.6
+        set pod's dns policy to ClusterFirstWithHostNet
```

Comment:
shall we not push that in the default cronjob on the [app](https://github.com/adfinis/openshift-etcd-backup)? and then bump it?

Reply:
Are https://github.com/adfinis/helm-charts/blob/main/charts/openshift-etcd-backup/templates/cronjob.yaml and https://github.com/adfinis/openshift-etcd-backup/blob/main/backup-cronjob.yaml kept in sync?

Reply:
https://github.com/adfinis/openshift-etcd-backup/pull/109

Reply:
Done, 1.9.7 released.